sale_payment_sheet/models/sale_payment_sheet.py

```diff
@@ -164,7 +163,6 @@ def button_confirm_sheet(self):
                     }
                 else:
                     vals_dic[key]["amount"] += amount_line
-                    vals_dic[key]["name"] += " {}".format(line.invoice_id.name)
```

Comment:
Better use vals_dic[key]["name"] += f" {line.invoice_id.name}" ??

Reply:
No, `name` mustn't be used under any circumstance, as it's the journal entry number. You have already put this text on other places where they are shown in the statement.